hoc/withTracker.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Libraries
+let GA;
+if (typeof window !== 'undefined') {
+  /* eslint-disable global-require */
+  GA = require('react-ga');
+  /* eslint-enable global-require */
+  const gaTrackingId = process.env.NODE_ENV === 'production' ?
+    process.env.GA_TRACKING_ID : 'UA-XXXXXXX-XX';
+  GA.initialize(gaTrackingId);
+}
+
+
+const withTracker = (Page, options = {}) => {
+  const trackPage = (page) => {
+    GA.event({
+      page,
+      category: page,
+      action: 'Navigation',
+      label: page,
+      ...options
+    });
+
+    GA.pageview(page);
```

Comment:
There is a little difference from the documentation of how to track a page and your code...
You are not setting the page like they do. Is there any reason?

![image](https://user-images.githubusercontent.com/8928965/30964700-d1d0d93a-a452-11e7-8323-2278e7609455.png)

Reply:
Now it follows doc instructions. We'll see how it works. Thanks!